tests/test_box_model.py

```diff
@@ -59,7 +59,7 @@ def get_auto_height(container: Size, parent: Size) -> int:
     box_model = get_box_model(
         styles, Size(60, 20), Size(80, 24), get_auto_width, get_auto_height
     )
-    assert box_model == BoxModel(Size(60, 20), Spacing(1, 2, 3, 4))
+    assert box_model == BoxModel(Size(54, 16), Spacing(1, 2, 3, 4))
```

Comment:
Just to be sure that I understand this correctly... The returned box model now shrinks the available space of a widget according to its margin, right? :slightly_smiling_face:

Reply:
yep